OneCommunityGlobal/HGNRest
# Imran_See_User_Management_Tab_Full_Functionality

# Description
Create "See User Management Tab (Full Functionality)" permission.
Make the "Other Links" -> "User Management" option appear/accessible with full functionality of User Management page (user is able to create, delete, and update users) for users who have the "See User Management Tab (Full Functionality)" permission.

## Related PRS (if any):
To test this backend PR you need to checkout the [#2478](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2478) frontend PR.
…

## Main changes explained:
- added new Permission 'userManagementFullFunctionality'

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Follow steps on the Frontend PR to test

## Note:
Include the information the reviewers need to know.
